test/syscalls/linux/udp_socket_test_cases.h

```diff
@@ -42,6 +39,9 @@ class UdpSocketTest
     }
   }
 
+  // The port to be be used on gvisor.
+  int TestPort;
```

Comment:
To follow naming conventions, this should be `test_port_`.

Reply:
Hi,
My test command is:
bblu@entos-thunderx2-04:~/gvisor/gvisor$ bazel test --test_tag_filters=runsc_ptrace //test/syscalls/...

There will always be a certain probability of failure in this case, even in my x86 machine.

Reply:
> To follow naming conventions, this should be `test_port_`.

Done.

Reply:
Yes. This is a workaround.
I think using random port is a good idea to reduce this probability.